components/style/gecko/wrapper.rs

```diff
@@ -387,6 +297,55 @@ impl<'le> GeckoElement<'le> {
         let extra_data = ParserContextExtraData::default();
         parse_style_attribute(value, &base_url, Box::new(StdoutErrorReporter), extra_data)
     }
+
+    fn flags(&self) -> u32 {
+        self.raw_node()._base._base_1.mFlags
+    }
+
+    fn raw_node(&self) -> &RawGeckoNode {
+        &(self.0)._base._base._base
```

Comment:
Surprisingly yes - weird interaction of newtype and underscore-prefixed identifiers: https://is.gd/T1Tqct